docs/src/administration_install_docker.md

```diff
@@ -17,6 +17,14 @@ mkdir -p volumes/pictrs
 sudo chown -R 991:991 volumes/pictrs
 ```
 
+Open up your `docker-compose.yml`, and make sure `LEMMY_EXTERNAL_HOST` for `lemmy-ui` is set to your correct host.
+
+```
+- LEMMY_INTERNAL_HOST=lemmy:8536
+- LEMMY_EXTERNAL_HOST=localhost:8536
```

Comment:
Based on the template above, this should be the domain, no? So I would put something like `your-domain.com` here.

Reply:
Ya, I'll add.